docs/identities/model/manage-identity-schema.mdx

```diff
@@ -1,51 +1,52 @@
 ---
 id: manage-identity-schema
-title: Manage identity schemas
-sidebar_label: Manage identity schemas
+title: Create & update identity schemas
 ---
 
-# Manage identity schemas
+## Creating identity schemas
```

Comment:
I think we can omit the headline or make it h1 :) Otherwise we have a H1 directly followed by a H2. WDYT?

Reply:
```suggestion
# Creating identity schemas
```